config/crd/bases/tests.testkube.io_scripts.yaml

```diff
@@ -131,12 +137,18 @@ spec:
                         description: if needed we can checkout particular path (dir
                           or file) in case of BIG/mono repositories
                         type: string
+                      token:
```

Comment:
Technically we don't need them in crd, because they are stored in secrets, so I remove them manually, because don't know how to skip generating token and username fields